client/src/App.js

```diff
@@ -39,7 +40,8 @@ const App = () => {
 							<Route exact path='/home' component={Home}/>
 							<Route exact path='/options' component={Options}/>
 							<Route exact path='/waiting-room/:roomId' component={WaitingRoom}/>
-							<Route exact path='/active-game/:roomId' component={ActiveGame}/>
+							<Route exact path='/active-game/:roomId' component={ArtistView}/>
+							<Route exact path='/active-game/:roomId' component={ObserverView}/>
```

Comment:
As is, the Router would never land on Artist View because the paths are the same. I think these two views would have to be on one page and that page would have to swap between the two.

Reply:
OK. Let's keep it for now, can move it to a void folder so it's not in the way, and once we figure out who that functionality works we can either use both pages or toggle between the two?